packages/application-shell/package.json

```diff
@@ -119,7 +119,7 @@
   "peerDependencies": {
     "@apollo/client": "3.x",
     "@testing-library/react": "12.x",
-    "@testing-library/react-hooks": "7.x",
+    "@testing-library/react-hooks": "8.x",
```

Comment:
This will likely cause a breaking change, thus bumping a major version.

If possible, we should keep it backwards compatible, so we allow both `7.x || 8.x`.

Reply:
Updated in this commit: 62476ca66a6d3b71e93166e08cf8c495834c5c2f